template/src/ontology/Makefile.jinja2

```diff
@@ -219,13 +219,19 @@ RELEASE_ASSETS = \
   $(REPORT_FILES){% endif %}
 
 .PHONY: all_assets
-all_assets: $(ASSETS)
+all_assets: $(ASSETS) check_rdfxml_assets
 
 .PHONY: show_assets
 show_assets:
 	echo $(ASSETS)
 	du -sh $(ASSETS)
 
+.PHONY: check_rdfxml_assets
+check_rdfxml_assets:
+	@for product in $(MAIN_PRODUCTS) ; do  \
```

Comment:
I wonder if this goal should dependent on $(MAIN_PRODUCTS). Remember that `prepare_release` moves those elsewhere.. alternatively, maybe in line with profile checking etc, we could use `check_rdfxml_%: %`?

Reply:
For now, the rule is a pre-requisite to `all_assets`, which means it is automatically called _after_ the assets are generated, and _before_ they are moved to the top-level directory by `prepare_release`. This seemed to me like a good place, but I am open to consider other options.

Reply:
All other PHONY goals can be run independently of any other goal - I would kind of like this to be generally the case. If my RDFXML is broken, and I am working on a fix in my edit file, I want to quickly run `sh run.sh  make check_rdfxml_assets` to ensure its fixed..

Reply:
So what about having:

* `check_rdfxml_%: %` (as you initially suggested), allowing to do a quick check on a arbitrary file (possibly not even a release product, e.g. if your -edit file is already a RDF/XML file and you want to check it without having to run the pipeline to produce the release files, you could call `sh run.sh make check_rdfxml_myont-edit.owl`), and
* `check_rdfxml_assets` dependent on `check_rdfxml_X.owl`, with _X_ being each RDF/XML product file

?

Reply:
Ok, sounds good!